sdk/appconfiguration/Azure.ApplicationModel.Configuration/tests/SessionRecords/ConfigurationLiveTests/AddExistingSetting.json

```diff
@@ -0,0 +1,144 @@
+{
+  "_entries": [
+    {
+      "RequestUri": "https://pakrym-azconfig-ui.azconfig.io/kv/key-1540471636?label=test_label",
```

Comment:
Should your name be replaced with something generic?

Reply:
I'm not sure how to do it. I want to strictly compare URIs and they are based on the connection string. I also don't want to build in too much knowledge about connection string <-> uri relationship into the recorder because it might not be true for other services.

Reply:
It should be in the service's sanitizer, no?

Reply:
I was just thinking about replacing all occurrences of  `pakrym` with something like `sample` or similar, including in the connection string.   I didn't mean to imply the need for more generalized behavior.

Reply:
It's easy to do in the simple example of AppConfig where you only need to sanitize connection string and URI but some services (KeyVault) in particular return the hostname as part of the body requiring the sanitizer to be way more complicated. I'm not sure there is a ton of benefit in stripping this information out. If we want to reduce diffs and make connection string less ugly we can have a set of well-known connection options we use for live tests in development time and session generation.